src/documentation/text/01_Preface/task-definition.tex

```diff
@@ -0,0 +1,6 @@
+Die vorliegende Seminararbeit thematisiert den Anwendungsfall eines Ticketverkaufs für Veranstaltungen.
+Kunden und Verwalter erhalten über spezielle Clients Zugriff auf eine Verwaltungskomponente, dessen Implementierung auf einem Aktorensystem basiert.
+Hierbei muss die Implementierung des Systems alle Anwendungsfälle erfüllen.
+Dafür müssen zunächst die Anforderungen an das System gesammelt werden, bevor eine Unterteilung in funktionale und nicht-funktionale Anforderungen stattfindet.
+Darauf aufbauend wird sowohl die Gesamtarchitektur als auch die Architektur der Teilsysteme entworfen.
```

Comment:
Die Architektur der Teilsysteme ist doch sowieso immer in der Gesamtarchitektur eingeschlossen und wird demnach auch entworfen, wenn die Gesamtarchitektur entworfen wird. Das andere wäre meines Wissens der Grobentwurf. Oder?
Vielleicht das entworfen durch ein näher beschrieben ersetzten.
```suggestion
Darauf aufbauend wird sowohl die Gesamtarchitektur als auch die Architektur der Teilsysteme näher beschrieben.
```

Reply:
Die Gesamtarchitektur wäre in unserem Fall das Client-Server-Modell. Es gäbe ja noch andere Modelle zur Auswahl. Hier könnte man darauf eingehen, wie Client- und Serverkomponente miteinander kommunizieren etc. Die Architektur der Teilsysteme bezieht sich dann explitizit auf die Architektur des Servers (Aktorensystem, etc.) und die Architektur des Clients (Module, etc.). 
`näher beschrieben` passt m.E. nicht ganz so gut, da der Leser an diesem Punkt des Textes ja noch garnichts über die Systeme weiß.